agentops/session.py

```diff
@@ -43,21 +46,52 @@ def set_session_video(self, video: str) -> None:
         """
         self.video = video
 
+    def add_metadata(self, key: str, value: Any) -> None:
```

Comment:
what kind of metadata are you thinking this would be used for? we currently have tag functionality. i'm open to building out this feature if there's a use case, but it would require a bit of dashboard and db work as well.